client/build.zig

```diff
@@ -3,7 +3,7 @@ const std = @import("std");
 // Although this function looks imperative, note that its job is to
 // declaratively construct a build graph that will be executed by an external
 // runner.
-pub fn build(b: *std.Build) void {
+pub fn build(b: *std.Build) !void {
```

Comment:
I'm not a huge fan of things named `a`, `b`, `c`, etc. Is this a Zig convention that I'm not aware or we can do better?

Reply:
It is conventional to name the build object `b` in your build functions, yes. It's even like that in the default `build.zig` you get when you `zig init` a new project.